components/form/LocationField.tsx

```diff
@@ -37,6 +37,26 @@ export const LocationField = React.forwardRef<RNView, LocationFieldProps>(({name
   const [mapReady, setMapReady] = useState<boolean>(false);
   const mapRef = useRef<MapView>(null);
 
+  const mapPanResponder = useRef(
+    PanResponder.create({
+      onStartShouldSetPanResponder: () => true,
+      onMoveShouldSetPanResponder: () => true,
+      onPanResponderRelease: (event, gestureState) => {
+        if (gestureState.dx != 0 && gestureState.dy != 0) {
+          return;
```

Comment:
I had a `console.log` here, but that's not really anywhere else in the repo. Should I add a log here using the `Logger` class, or leave it as is?